sdk/nodejs/api/client.gen.ts

```diff
@@ -289,6 +291,7 @@ export class Container extends BaseClient {
 
   /**
    * Retrieve a directory at the given path. Mounts are included.
+   *
```

Comment:
The codegen is generating unnecessary extra line. Comment valid for the whole file.

Reply:
We decided to remove any extra line, to be idiomatic with jsdoc and avoid this bikeshedding again: https://jsdoc.app/about-getting-started.html